### PR TITLE
Moved SendStatus into the Timestamp component

### PR DESCRIPTION
**Pull request by tdurnford**
### Change Log
-  Fixes [#2747](https://github.com/microsoft/BotFramework-WebChat/issues/2747). Moved `SendStatus` into the `Timestamp` component and removed the `sendStatus` style set, by [@tdurnford](https://github.com/tdurnford) in PR [#2735](https://github.com/microsoft/BotFramework-WebChat/pull/2735)

### Description
Moved the `SendStatus` component from the `StackedLayout` and `CarousalLayout` components to the `TimeStamp` component reducing redundant code across the layouts.

### Additional Context
Simplifies code changes for #2722

---

-  [ ] Testing Added

### CODE REVIEW

**Comment by corinagum** [comment]
Hmm, changelog, just in case?

File: packages/component/src/Activity/SendStatus.js [hunk 1]
```diff
@@ -31,10 +31,15 @@ const connectSendStatus = (...selectors) =>
...
 
 const SendStatus = ({ activity }) => {
-  const [{ sendStatus: sendStatusStyleSet }] = useStyleSet();
+  const [{ sendStatus: sendStatusStyleSet, timestamp: timestampStyleSet }] = useStyleSet();
   const focusSendBox = useFocusSendBox();
   const postActivity = usePostActivity();
 
+  sendStatusStyleSet &&
+    console.warn(
+      'Web Chat: styleSet.sendStatus is being deprecated. Please use styleSet.timestamp. This deprecation migration will be removed on or after December 17, 2021.'
```
Comment (corinagum) [inline packages/component/src/Activity/SendStatus.js]: Hmm I'm not sure about the naming here. timestamp can be considered a send status, but send status is not a timestamp. Should we flip the variable names, or come up with a new container name?

File: packages/component/src/Activity/SendStatus.js [hunk 2]
```diff
@@ -31,10 +31,15 @@ const connectSendStatus = (...selectors) =>
...
+    );
+
   // TODO: [P4] Currently, this is the only place which use a templated string
   //       We could refactor this into a general component if there are more templated strings
```
Comment (corinagum) [inline packages/component/src/Activity/SendStatus.js]: What does this mean @compulim?

Comment (compulim) [reply packages/component/src/Activity/SendStatus.js]: The templated string is referring to localization. "Send failed. {Retry}."

In order to get the hyperlink localized correctly, we localize it in 2-pass:

1. First, it localize `Retry` as `再送`
1. Then, it put the `再送` inside this `送信できませんでした。{Retry}。` and localize it again.

Template is referring to `{Retry}`.

We can easily write a better localizer that do nested localization, so it handle more strings than `{Retry}`. Or, we localize it using Markdown.

File: packages/component/src/Activity/SendStatus.js (hunk 1 above)
Comment (compulim) [reply packages/component/src/Activity/SendStatus.js]: I prefer flip too. Send status is broader than timestamp. 😉

File: packages/component/src/Activity/SendStatus.js (hunk 2 above)
Comment (corinagum) [reply packages/component/src/Activity/SendStatus.js]: Sounds like we may be contracting out localization help for Web Chat in the near future, so maybe we can take this discussion to next Applications standup. Thoughts?

File: packages/component/src/Activity/SendStatus.js (hunk 1 above)
Comment (tdurnford) [reply packages/component/src/Activity/SendStatus.js]: You guys are the worst

**Pull request closed without merging**